client/mass/charts.js

```diff
@@ -43,8 +43,7 @@ class MassCharts {
 		) {
 			// force to show a dictionary chart button
 			// TODO: may want the server to decide this, and as defined for a dataset
-			if (state.vocab.dslabel == 'ProfileFull' || state.vocab.dslabel == 'ProfileAbbrev')
-				state.supportedChartTypes.push(...appState.termdbConfig.allowedChartTypes)
+			if (state.vocab.dslabel == 'profile') state.supportedChartTypes.push(...appState.termdbConfig.allowedChartTypes)
```

Comment:
delete `if profile` and change to include allowedChartTypes if it is present

Reply:
I dont think it is that simple, we may need to discuss this change with @siosonel.